src/style.css

```diff
@@ -0,0 +1,419 @@
+}
+
+header nav ul {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+}
+
+header nav ul li {
+  padding-left: 1em;
+  cursor: pointer;
+}
+
+header nav ul #meals {
+  margin: 0 1em 0 -3.5em;
+  font-weight: 500;
+  font-size: 1.1em;
+}
+
+footer {
+  display: flex;
+  place-items: center;
+  position: relative;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  background-color: white;
+  height: 5em;
+  box-shadow: var(--box-shadow);
+}
+
+footer h2 {
+  margin-left: 1em;
+  font-size: 1.2em;
+  font-weight: 500;
+}
+
+.home {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  margin: 8em 0;
+  width: 100%;
+}
+
+.cont-popup p {
+  margin-right: 2em;
+  font-size: 1.2em;
+}
+
+.meal-popup p {
+  text-align: left;
+  margin: 1em 1em;
+  line-height: 2em;
+  font-size: 1.1em;
+  font-weight: 500;
+}
+
+.meal {
+  display: grid;
+  grid-template-rows: auto 25% 25%;
+  margin: 0 1em 5em 0;
+  width: 20em;
+  height: 33em;
+  background-color: white;
+  border-radius: 0.5em;
+  box-shadow: var(--box-shadow);
+}
+
+.meal .div-like {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin: auto 0;
+  width: 100%;
+}
+
+.meal .div-like p {
+  text-align: left;
+  margin-left: 1em;
+  font-size: 1.2em;
+  font-weight: 500;
+}
+
+.div-heart {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 10em;
+}
+
+.meal .div-like .div-heart .fa-heart {
+  margin-left: 1em;
+  font-size: 1.7em;
+  cursor: pointer;
+  color: var(--orange);
+}
+
+.meal .div-like .div-heart .fa-heart:hover {
+  color: rgb(255, 165, 0, 0.7);
+}
+
+.meal .div-like .div-heart .fa-heart:active {
+  color: red;
+}
+
+.meal .div-like .div-heart label {
+  font-size: 1em;
+  color: var(--black);
+  margin-left: 1.2em;
+}
+
+.div-img {
+  place-self: center;
+  margin: 1em auto;
+}
+
+.meal img {
+  width: 17em;
+  border-radius: 0.5em;
+}
+
+button {
+  text-align: center;
+  background-color: var(--black);
+  color: white;
+  width: 8em;
+  height: 2.5em;
+  font-weight: 500;
+  font-family: "Poppins", sans-serif;
+  font-size: medium;
+  place-self: center;
+  cursor: pointer;
+  margin-bottom: 3em;
+  border: none;
+  border-radius: 0.5em;
+  box-shadow: var(--box-shadow);
+}
+
+.random h2 {
+  margin: 3em 0 -3em 0;
+  font-size: 1.7em;
+  text-align: center;
+}
+
+.meal-popup {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  overflow-y: scroll;
+  height: 100%;
+  z-index: 999;
+  box-shadow: var(--box-shadow);
+  background-color: rgb(241, 240, 240);
+}
+
+.meal-popup h2 {
+  text-align: center;
+  font-weight: bold;
+  font-size: 2.5em;
+}
+
+.meal-popup h4 {
+  text-align: left;
+  font-weight: bold;
+  font-size: 1.2em;
+  margin: 1em 1em;
+}
+
+/*
+.meal-popup-close-icon {
+  font-size: 32px;
+  color: black;
+  position: fixed;
+  top: 18px;
+  right: 28px;
+}
+*/
```

Comment:
[OPTIONAL] Do not leave unused codes as comments in your code base since it makes it bulky and slows down performance :+1: